connectors/twitter-mention-connector/src/main/java/io/syndesis/connector/twitter/springboot/TwitterMentionConnectorConfigurationCommon.java

```diff
@@ -1,6 +1,7 @@
 package io.syndesis.connector.twitter.springboot;
 
 import javax.annotation.Generated;
+import io.syndesis.connector.twitter.TwitterMentionComponent;
```

Comment:
It looks this import just came alone without use?

Reply:
yeah, those are generated by camel-package-maven-plugin (IIRC), I'll modify it locally and update